builtin/game/item.lua

```diff
@@ -139,6 +139,19 @@ function core.dir_to_wallmounted(dir)
 	end
 end
 
+function core.wallmounted_to_dir(wallmounted)
+	-- table of dirs in wallmounted order
+	return ({[0]={x=0, y=1, z=0},
+					{x=1, y=-1, z=0},
```

Comment:
I've originally meant the configuration of the x, y, and z. I don't really know much about the wallmounted format, but that single value seems outside of the system.

But yeah, indentation should be fixed as well. Best make it one more than the return.